h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -849,12 +848,11 @@ public Persist getPersistForURI(URI uri) {
    * @return boolean
    */
   public boolean isFileAccessDenied(String path) {
+    if (isHdfsPath(path) || isGcsPath(path) || isS3Path(path)) {
```

Comment:
what about nfs path? would't it be safer to check it just on `file:` schema instead of explicitly specifying the list of schemas that are ingored ?